docs/api/baseplate/clients/sqlalchemy.rst

```diff
@@ -52,6 +52,14 @@ request::
 Configuration
 -------------
 
+.. warning::
+
+   :py:class:`sqlalchemy.pool.QueuePool` does not handle `ServerTimeout`
```

Comment:
Is there a code example of how to do this the right way vs wrong way? That'd be great to include here

(generally I prefer it when saying "Don't do it this way" to guide people through the right way of doing it (though the starting pointers of "do this instead" helps a lot))